### PR TITLE
blog: announcing Ember TypeScript Core Team

### PR DESCRIPTION
**Pull request by chriskrycho**
## What it does

Tells the world that we’re adding a new Core team! (For lookers-on: this has already gone through a review round with Steering.) 🎉 

See also ember-learn/ember-website#922

### CODE REVIEW

File: content/typed-ember-is-now-the-ember-type-script-core-team.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Typed Ember is now the Ember TypeScript Core Team!
+authors:
+  - chris-krycho
+date: 2023-03-09T18:40:00.000Z
```
Comment (jaredgalanis) [inline content/typed-ember-is-now-the-ember-type-script-core-team.md]:
```suggestion
date: 2023-03-010T18:40:00.000Z
```

File: content/typed-ember-is-now-the-ember-type-script-core-team.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - '2022'
+  - typescript
+  - announcement
+  - community
+
+---
+
+Last Friday, the Ember project merged [RFC #0724: Official TypeScript Support](https://emberjs.github.io/rfcs/0724-road-to-typescript.html), committing to make TypeScript an officially-supported language for Ember.js. To make that a reality, we also need to have a team dedicated to maintaining Ember’s types and its TypeScript compiler integrations.
+
+Happily, we already have a group of people who do that! For the last five years, Ember has had community-driven TypeScript support, led by the past and present members of the Typed Ember team.
```
Comment (jaredgalanis) [inline content/typed-ember-is-now-the-ember-type-script-core-team.md]:
```suggestion
Happily, we already have a group of people who do that! For the last five years, Ember has had community-driven TypeScript support, led by the past and present members of the Typed Ember team.
```

**Review by jaredgalanis — APPROVED** [review]
❤️ just a few comments, but otherwise lgtm!

File: content/typed-ember-is-now-the-ember-type-script-core-team.md (hunk 1 above)
Comment (chriskrycho) [reply content/typed-ember-is-now-the-ember-type-script-core-team.md]:
```suggestion
date: 2023-03-10T12:30:00.000Z
```

**Pull request merged**